### PR TITLE
Resolve notes

### PR DESCRIPTION
**Pull request by karollayna**

### CODE REVIEW

**Comment by karollayna** [comment]
I was trying to resolve 8notes:

1. Consider adding the maintainer's ORCID iD in 'Authors@R' - not done
2. Avoid 1:...; use seq_len() or seq_along() - done
3. Avoid 'cat' and 'print' outside of 'show' methods - done for 1 of 3
4. The recommended function length is 50 lines or less. There are 7 functions greater than 50 lines. - not done
5. Consider adding runnable examples to man pages that document exported objects. - added one example
6. Consider adding a NEWS file, so your package news will be included in Bioconductor release announcements. - done
7. Consider shorter lines; 148 lines (5%) are > 80 characters long. - limited
8. Consider multiples of 4 spaces for line indents; 618 lines (23%) are not. - not done

File: NEWS.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## 0.99.0 (05.02.2024)
```
Comment (gladkia) [inline NEWS.md:1]:
```suggestion
## 0.99.2 (2024-02-09)
- fix some Bioconductor NOTEs

## 0.99.1 (2024-02-05)
- fix some Bioconductor NOTEs

## 0.99.0 (2024-02-04)
- make the first Bioconductor release

```
```suggestion
## 0.99.0 (05.02.2024)
```

File: NEWS.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+- added NEWS file
```
Comment (gladkia) [inline NEWS.md:2]:
```suggestion
```

File: NEWS.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+## 0.99.0 (05.02.2024)
+- added NEWS file 
+
+## 0.99.2
```
Comment (gladkia) [inline NEWS.md:4]:
```suggestion
```

File: NEWS.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+- reformatted code
```
Comment (gladkia) [inline NEWS.md:5]:
```suggestion
```

**Comment by gladkia** [comment]
GJ!

**Review by gladkia — APPROVED** [review]

**Pull request merged**